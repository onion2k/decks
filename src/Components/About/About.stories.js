import React from 'react';
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import About from '../Components/About';

storiesOf('About', module)
  .add('Basic', () => (
    <About />
  ));
