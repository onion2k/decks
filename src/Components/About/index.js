import React, { Component } from "react";
import "./About.css";

class About extends Component {
  render() {
    return (
      <div className="About">
        <p>A video playlist manager with a record player UI.</p>

        <p>
          <a href="/#v_yTphvyiPU,Rs38lKxmtI4,BuVJEn9wk9Y,e4Ao-iNPPUc,FALYmqt-7TQ,a3Z4RWZa9WA">
            Playlist
          </a>
        </p>
      </div>
    );
  }
}

export default About;
