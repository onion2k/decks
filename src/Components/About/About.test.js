import React from "react";
import renderer from "react-test-renderer";
import { MemoryRouter } from 'react-router-dom'

import About from ".";

describe("About", () => {
  const onClick = jest.fn();

  it("renders properly", () => {
    const about = renderer
      .create(
        <MemoryRouter>
          <About />
        </MemoryRouter>
      )
      .toJSON();
    expect(about).toMatchSnapshot();
  });
});
