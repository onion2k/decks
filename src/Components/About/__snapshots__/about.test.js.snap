// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders properly 1`] = `
<div
  className="About"
>
  <p>
    A video playlist manager with a record player UI.
  </p>
  <p>
    <a
      href="/#v_yTphvyiPU,Rs38lKxmtI4,BuVJEn9wk9Y,e4Ao-iNPPUc,FALYmqt-7TQ,a3Z4RWZa9WA"
    >
      Playlist
    </a>
  </p>
</div>
`;
